frontend/das_haus/src/util/routeProtection.js

```diff
@@ -0,0 +1,13 @@
+import {redirect} from "react-router-dom";
+
+export function routeProtectionLoader() {
+  let isAuthenticated = localStorage.getItem("token");
+  if (!isAuthenticated) {
+    return redirect("/auth?mode=login");
+  }
+  else {
+    return null;
+    
```

Comment:
remove these 2 new lines here

Reply:
This is how react router works - please read the documentation